examples/discovery-search-app/cypress/integration/passages/spec.ts

```diff
@@ -24,7 +24,7 @@ describe('Passage Results', () => {
     });
 
     it('SearchResults displays ONLY the first passage text of the results that have passages', () => {
-      cy.get('.bx--search-result__content-wrapper__body')
+      cy.get('div[data-testid="search-result-element-body-passage"]')
```

Comment:
we could bring in the https://github.com/testing-library/cypress-testing-library to provide a native `getByTestId`

Reply:
Does it make sense to use snake_case here to align in how our `data-test-id`s are being formatted in Tooling? Just thinking of some ways we could maintain consistency to minimize the friction when switching between the projects